specifyweb/specify/tree_extras.py

```diff
@@ -10,7 +10,7 @@
 
 from specifyweb.businessrules.exceptions import TreeBusinessRuleException
 
-from  .auditcodes import TREE_MERGE, TREE_SYNONYMIZE, TREE_DESYNONYMIZE
+from  .auditcodes import TREE_BULK_MOVE, TREE_MERGE, TREE_SYNONYMIZE, TREE_DESYNONYMIZE
```

Comment:
Currently, we are not using `TREE_BULK_MOVE`. 

Be sure to use the `mutation_log` function once the Preparations are successfully moved from one Storage node to another to insert the Action into the Audit Log (there are examples of this function for other Tree Actions in this file).
https://github.com/specify/specify7/blob/b331ce9c552e90d86fc4811a7089fab6ac29c07e/specifyweb/specify/tree_extras.py#L235-L237

https://github.com/specify/specify7/blob/b331ce9c552e90d86fc4811a7089fab6ac29c07e/specifyweb/specify/tree_extras.py#L286-L287